docs/data/versions.yaml

```diff
@@ -0,0 +1,7 @@
+params:
```

Comment:
Before we check in, I believe we'll want to make sure 2.0 is removed from this list. I don't believe we want to turn on our 2.0 docs until we're ready to launch the beta (we're actively working on trimming the API surface for it between now and release).

I think we just want the 1.0 docs for now.

@rohandubal @sashasimic interested in your opinions on this :)

Reply:
Agreed, we should work to stage this change (if possible) or test in an alternate manner and keep a separate PR closer to the launch.

Reply:
Understood. Remove 2.0 for now

Reply:
Agreed. We can toggle it when 2.0 rc is released.

Reply:
Updated. Seems like index.js takes care of multi-version flag. It's off now